### DISCUSSION TITLE
USD import UV issue

### DISCUSSION THREAD

**Discussion by vfxthread** [Autodesk/maya-usd#1031] [Q&A]
Hi 
I am using this plugin in maya2020 for my usd export
https://github.com/Autodesk/maya-usd...ses/tag/v0.6.0

The issue is that if I were to export with "animation data" off,
and then if I go to another maya scene and file > import the result , I can see UVs.

But if i were to export with "animation data" ON
and then if I go to another maya scene and file > import the result, I can not see any UVs.

why is this happening?

**Comment by dgovil** [comment]
Exporting with a time range results in the UV attributes being recorded as time sampled, even if they're a single time sample. 

The importer currently doesn't import animated mesh values. See https://github.com/Autodesk/maya-usd/issues/973

The next release of the plugin should have an option called staticSingleSamples that will convert them to a non time sampled value instead.

It won't solve the issue but it will make it easier to workaround.

**Reply by vfxthread** [reply]
Thanks,

just to understand more clearly,
as you said, in maya Importer currently doesn't import animated mesh values.
But the "usd stage import" in maya does import with textures.. so i guess the uv information do exist in the usd but the
importer is not able to bring it in as uvs are recorded as time sampled, even if they're a single time sample.

(I am trying to import the same character using the usd stage in unreal but in there also i get uvs and textures for static model but textures don't work on the animated model.)

**Comment by dgovil** [comment]
Yes, the data is all there. When brought in as a stage, it's not being
converted to Maya native data and is being served directly from the USD
runtime. When imported it's converting the USD to Maya data, but that means
there's various things that support hasn't been added for yet.

**Reply by vfxthread** [reply]
Thanks, can you also tell me about the rigs?
Does the plugin support exporting the rig along with the charecter?
